recipe/meta.yaml

```
@@ -213,22 +211,21 @@ outputs:
   - name: nvshmem4py
     version: {{ nvshmem4py_version }}
     build:
-      ignore_run_exports_from:    # [(cuda_compiler_version or "").startswith("12")]
-        - {{ compiler("cxx") }}   # [(cuda_compiler_version or "").startswith("12")]
-        - cuda-cudart-dev         # [(cuda_compiler_version or "").startswith("12")]
+      ignore_run_exports_from:
+        - {{ compiler("cxx") }}
+        - cuda-cudart-dev
       ignore_run_exports:
         - cuda-version
       string: "py{{ py }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}"
       skip: true  # [py<310 or py>313]
-      skip: true  # [not (cuda_compiler_version or "").startswith("12")]
+      skip: true  # [not (cuda_compiler_version or "").startswith(("12", "13"))]
```

This could simply be
```suggestion
      skip: true  # [cuda_compiler_version != "None"]
```
as there are no other values than 12.x or 13.x on the horizon for ~years.

---

Thanks! I agree